docs/cub/tuning.rst

```diff
@@ -82,15 +114,29 @@ a :code:`nvbench::type_list`. For more details on the benchmark signature, take
 
   template <typename T, typename OffsetT>
   void algname(nvbench::state &state, nvbench::type_list<T, OffsetT>)
-  {
+  {...}
+
+.. @giannis: not sure if Policy Hub should be part of "Authoring Benchmarks" since it's attached to the Dispatch Layer. Policy Hub
+.. is not supposed to be used only when we run benchmarks, but in general when a primitive is invoked with specific compile time params.
+
+Before proceeding further with the benchmark authoring it is imperative for the user to understand the Policy Hub mechanism.
```

Comment:
suggestion: user is actively reading this document, doesn't make sense to address them as third party:

```suggestion
Before proceeding further with the benchmark authoring it is imperative to understand the Policy Hub mechanism.
```